projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-wpcom-offline-subscription-service.php

```diff
@@ -44,12 +52,35 @@ public static function available() {
 	public function visitor_can_view_content( $valid_plan_ids, $access_level ) {
 		/** This filter is already documented in projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php */
 		$subscriptions = apply_filters( 'earn_get_user_subscriptions_for_site_id', array(), wp_get_current_user()->ID, $this->get_site_id() );
-		if ( empty( $subscriptions ) ) {
-			return false;
-		}
 		// format the subscriptions so that they can be validated.
-		$subscriptions = self::abbreviate_subscriptions( $subscriptions );
-		return $this->validate_subscriptions( $valid_plan_ids, $subscriptions );
+		$subscriptions      = self::abbreviate_subscriptions( $subscriptions );
+		$is_paid_subscriber = $this->validate_subscriptions( $valid_plan_ids, $subscriptions );
+
+		return $this->user_has_access( $access_level, true, $is_paid_subscriber, $this->post_id );
```

Comment:
It would be nice make `true` a variable for ease of readability

Reply:
You should use PHPStorm ;)

<img width="949" alt="Screenshot 2022-11-18 at 11 41 27" src="https://user-images.githubusercontent.com/790558/202693763-909f22d4-f9e4-41fd-9328-afd5ae2ba7dd.png">

Reply:
There is a plugin for VS Code:
https://marketplace.visualstudio.com/items?itemName=BobbyZrncev.intellij-parameter-hints